lib/Zonemaster/Backend/Config.pm

```diff
@@ -185,32 +185,84 @@ sub PollingInterval {
     return $self->{cfg}->val( 'DB', 'polling_interval' );
 }
 
+
+=head2 MaxZonemasterExecutionTime
+
+=head3 INPUT
+
+'max_zonemaster_execution_time' from [ZONEMASTER] section in ini file.
```

Comment:
Maybe link to the property in Configuration.md?

Reply:
Good suggestion, but those links will be broken until the updated Configuration.pm has been merged to master branch. I guess that is OK.